src/UnityExplorer.csproj

```diff
@@ -354,4 +354,9 @@
       <Private>False</Private>
     </Reference>
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="..\UniverseLib\src\UniverseLib.csproj">
+      <Link>UniverseLib.csproj</Link>
+    </Content>
+  </ItemGroup>
```

Comment:
Is this change necessary?

Reply:
This was done automatically by my IDE so references weren't broken while programming. It can probably be removed and still be built fine.